src/models/thread.ts

```diff
@@ -47,6 +48,9 @@ export class Thread extends TypedEventEmitter<ThreadEvent> {
 
     private reEmitter: ReEmitter;
 
+    private _lastReply: MatrixEvent;
+    private _replyCount = 0;
```

Comment:
I believe `_` prefix is discouraged for our TS files (unless there's a getter with the same name), since `private` should be enough.

```suggestion
    private lastReply: MatrixEvent;
    private replyCount = 0;
```